### ISSUE TITLE
Customized extent and znear zfar recomputation

### ISSUE THREAD

**Issue by agrignard** [VCityTeam/UD-Viz#345]
With a custom extern such as this one on Gratte Ciel District (smaller than the "default" one)

`````
  "extents": {
    "min_x": "1845869.7865",
    "max_x": "1846369.7865",
    "min_y": "5175547.8019000003",
    "max_y": "5176547.8019000003"
  },
``````

This is how the display looks like:

<img width="1357" alt="Screenshot 2022-04-13 at 14 52 08" src="https://user-images.githubusercontent.com/3928502/163184171-953965a4-ebcc-4992-b805-3276564bc7e7.png">

**Comment by agrignard** [comment]
This one might be related to #199

**Comment by valentinMachado** [comment]
fix in 2.37.4 version
